### ISSUE TITLE
Training button greyed out (stops training) after making edits

### ISSUE THREAD

**Issue by RussellSPG** [samtecspg/articulate#1061]
### Issues
When I am training an agent, then decide to make an edit to anything such as an action/response/keyword etc, the training button is greyed out after saving.

Shutting down the agent from gitbash and restarting fixes this issue.

Master
FireFox

**Comment by wrathagom** [comment]
so the agent is already training and you make a change? Does it un-grey after the previous training finishes?

**Comment by RussellSPG** [comment]
It goes back to training... but its stuck.

**Comment by RussellSPG** [comment]
If I hit the refresh browser, then it goes back to grey with "training needed"

**Comment by RussellSPG** [comment]
After shutting down from gitbash and restarting sometimes it will just show a button with "training error". Then I train again, and its fine.

**Comment by wrathagom** [comment]
I need to try and reporoduce this one, It actually sounds like expected behavior until you forced restarted things, which hosed it. But certainly could be broken.

**Comment by wrathagom** [comment]
@dakavisa I was able to re-produce this one. I added a sayings and clicked trying. While it was training I deleted the saying. A few observations:

- A tooltip popped up saying "Another agent is training". That's an okay thing, but I couldn't get the tooltip to come back up after that.
- When I first did it the training animation stopped, but the button was just greyed out. This makes sense since I can't click train again until the previous training completed.
- Then the training animation came back...
